lib/setup.coffee

```diff
@@ -135,9 +135,16 @@ module.exports = (app) ->
 
   console.log 'Mounting apps...'
 
+  console.log 'Watching for changes...'
```

Comment:
Not needed (prod isn't going to watch for changes)

Reply:
oh i see, i thought it was informative for the dev environment.